src/selectedtests/task_mappings/update_task_mappings.py

```diff
@@ -1,16 +1,84 @@
 """Methods to update task mappings for a project."""
+from typing import Any, Dict, List
+
 import structlog
 
 from evergreen.api import EvergreenApi
+from pymongo import ReturnDocument, UpdateOne
+from pymongo.errors import BulkWriteError
 
 from selectedtests.datasource.mongo_wrapper import MongoWrapper
+from selectedtests.helpers import create_query
 from selectedtests.project_config import ProjectConfig
 from selectedtests.task_mappings.create_task_mappings import generate_task_mappings
 from selectedtests.task_mappings.version_limit import VersionLimit
 
 LOGGER = structlog.get_logger()
 
 
+def update_task_mappings_tasks(
+    tasks: List[Dict[str, Any]], task_mapping_id: Dict[str, Any], mongo: MongoWrapper
+) -> None:
+    """
+    Update task in the task mappings tasks collection.
+
+    :param tasks: A list of tasks.
+    :param task_mapping_id: The containing task_mapping identifier.
+    :param mongo: An instance of MongoWrapper.
+    """
+    operations = []
+    for task in tasks:
+        query = create_query(task, mutable=["flip_count"])
+        query = dict(**query, **task_mapping_id)
+
+        update_test_file = UpdateOne(
+            query, {"$inc": {"flip_count": task["flip_count"]}}, upsert=True
+        )
+        operations.append(update_test_file)
+
+    try:
+        result = mongo.task_mappings_tasks().bulk_write(operations)
+        LOGGER.debug("bulk_write", result=result.bulk_api_result, parent=task_mapping_id)
+    except BulkWriteError as bwe:
```

Comment:
Can we add a test for the `BulkWriteError` test case?

Reply:
Agreed that it might not make sense to test log messages. Here, since the function has no return value, there's not much to test besides logs, which it makes it hard to test. I think what you added looks good.